integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/support/ws/smartmetering/bundle/SetPushSetupAlarmRequestBuilder.java

```diff
@@ -12,19 +12,30 @@
 import static org.opensmartgridplatform.cucumber.core.ReadSettingsHelper.getString;
 
 import java.math.BigInteger;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import org.opensmartgridplatform.adapter.ws.schema.smartmetering.bundle.SetPushSetupAlarmRequest;
+import org.opensmartgridplatform.adapter.ws.schema.smartmetering.common.ObisCodeValues;
+import org.opensmartgridplatform.adapter.ws.schema.smartmetering.configuration.PushObject;
 import org.opensmartgridplatform.adapter.ws.schema.smartmetering.configuration.PushSetupAlarm;
 import org.opensmartgridplatform.cucumber.platform.smartmetering.PlatformSmartmeteringKeys;
 
 public class SetPushSetupAlarmRequestBuilder {
 
   private static final String DEFAULT_HOST = "localhost";
   private static final int DEFAULT_PORT = 9598;
+  private static final String DEFAULT_CLASS_IDS = "1,40,1";
+  private static final String DEFAULT_OBIS_CODES = "0.0.96.1.1.255,0.1.25.9.0.255,0.0.97.98.0.255";
```

Comment:
I would expect OBIS code 0.0.97.98.1.255 somewhere in this code and test because that is the cause for this functionality?

Reply:
I've added an additional test for an SMR5.1 meter to test settings this object as well

Reply:
I've added an additional test for an SMR5.1 meter to test settings this object as well